axelrod/strategies/ann.py

```diff
@@ -169,62 +172,40 @@ class EvolvedANN(ANN):
-                       -1.927322672845826, 0.0014584395475859544, -8.629826916169318, 22.43281153854352,
-                       87.10895591188721, -0.22253937914423294, -233.06796470563208, -620.4917481128365,
-                       -1.8253699204909606,-0.0030318160426064467, -77.25818476745101, -2057.311059352977,
-                       3.764204074005541, -47.47629147374066, 233.16096124330778, -160721.96744375565,
-                       -278292.9688140893, -2.093640525920404, -142886.66171202937, 53.64449245132945,
-                       12.5162147724691, -207.75462390139955, 132167.659160016, 21.197418541051732,
-                       83979.45623573882, -49.47558832987566, 0.05242625398046057, -842.1484416713075,
-                       -0.1581049310461208, 2359.2124343564096, 1170.5147830681053, -847999.9145843558,
-                       -0.8053911061885284, -5363.722820739466, 171.58433274294117, -724.7468082647013,
-                       2500359.853524033, 1595.3955511798079, -4.254009123616706, -171.12968391407912,
-                       -32.30624102753424, -558.412338112568, -234.29754199019308, -18768.34057250429,
-                       8338.792126484348, -0.18593140210730602, -7.758804964874875, 0.39736677884665267,
-                       547.0567585452197, 1.1969366369973133, 0.4861465741177498, -51.19319208716985,
-                       12.775051406025534, -0.09185362260212569, 22.08417300332754, -5090.013231748707,
-                       -0.814394991797045, 1.1534025840023847, 8.390439959276764, -0.02227253403481858,
-                       0.14162040507921927, -0.011508263843203926, 0.22372493104861083, 0.7754713610627112,
-                       0.1044033140236981, -4.377055307648915, -41.898221495326574, -18656.755601828827,
-                       -134.56719406539244, -2405.8148785743474, 16864.049985157206, -0.5124682025216784,
-                       14521.069005125159, -10.740782200739309, 18756.807715014013, -1723.9353962656946,
-                       87029.99828299093, 5.7383786020894195e-05, 4762.960401619296, 0.7331769713238158,
-                       -308.5673034493341, 85.29725765515369, 0.4268843538235295, -0.17788805472511407,
-                       -1.1727033611646802, 7578.6822604990175, 0.5124673187864222, 0.1595627909684813,
-                       -145.93742731401096, -2954.234440189563, 0.009672881359732015, 106.4646644917487,
-                       -0.050606976105730346, 2.3904047264403596, -4.987645640997455, -43.22984692765006,
-                       -36.177108409134966, -0.3812547430698569, -2959.4921368963633, -1.8635802741029985,
-                       0.020513128847167047, -0.9179124323385958]
-
-        (i2h, h2o, bias) = split_weights(
-            weights,
-            input_values,
-            hidden_layer_size
-        )
-        ANN.__init__(self, i2h, h2o, bias)
+        num_features, num_hidden, weights = nn_weights['1']
```

Comment:
Could we change the `1` in the dataset and here to be `10`? This would be for readability and corresponding to the size of the hidden layer.